R/mobr_boxplots.R

```diff
@@ -231,59 +351,38 @@ get_mob_stats = function(mob_in,
       out$samples$S_rare <- list()
       
       n_rare_samples <- floor(n_rare_samples)
-      if (is.na(n_rare_samples)){   
+      if (!is.numeric(n_rare_samples) | is.na(n_rare_samples)){   
```

Comment:
The same comment of applying `is.na()` to vector also applies here. Or maybe I'm misunderstanding what you are trying to do?